ble/GapAdvertisingData.h

```diff
@@ -345,6 +431,27 @@ class GapAdvertisingData
         return (uint16_t)_appearance;
     }
 
+    /**
+     * Search advertisement data for field.
+     * Returns pointer to field if found, NULL otherwise.
+     */
+    uint8_t* findField(DataType_t type) {
```

Comment:
I'm using this function when scanning for peripherals. Should we put it somewhere else?